index.html

```diff
@@ -273,132 +273,141 @@ <h2>About Me</h2>
+              rows="5"
               placeholder="Enter text here"
               class="contact text"
+              maxlength="500"
+              required
             ></textarea>
             <div class="submit">
-            <input type="submit" value="Get In Touch" class="btn-submit">
+              <input type="submit" value="Get In Touch" class="btn-submit" />
             </div>
           </form>
         </section>
 
         <section class="footer">
           <div class="footer-socials">
-             <ul>
-                <li>
-                  <a href="https://github.com/Otegaa"
-                    ><img
-                      src="./assets/nav-icons/Footer-icon1.svg"
-                      alt="github"
-                      class="footer-icon"
-                  /></a>
-                </li>
-                <li>
-                  <a href="https://www.linkedin.com/in/blessing-ekiugbo/"
-                    ><img
-                      src="./assets/nav-icons/Footer-icon2.svg"
-                      alt="linkedIn"
-                      class="footer-icon"
-                  /></a>
-                </li>
-                <li>
-                  <a href="https://www.instagram.com/o_tegaaa/"
-                    ><img
-                      src="./assets/nav-icons/Footer-icon3.svg"
-                      alt="instagram"
-                      class="footer-icon"
-                  /></a>
-                </li>
-                <li>
-                  <a href="https://twitter.com/O_tegaaa"
-                    ><img
-                      src="./assets/nav-icons/Footer-icon4.svg"
-                      alt="twitter"
-                      class="footer-icon"
-                  /></a>
-                </li>
-              </ul>
+            <ul>
+              <li>
+                <a href="https://github.com/Otegaa"
+                  ><img
+                    src="./assets/nav-icons/Footer-icon1.svg"
+                    alt="github"
+                    class="footer-icon"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/blessing-ekiugbo/"
+                  ><img
+                    src="./assets/nav-icons/Footer-icon2.svg"
+                    alt="linkedIn"
+                    class="footer-icon"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.instagram.com/o_tegaaa/"
+                  ><img
+                    src="./assets/nav-icons/Footer-icon3.svg"
+                    alt="instagram"
+                    class="footer-icon"
+                /></a>
+              </li>
+              <li>
+                <a href="https://twitter.com/O_tegaaa"
+                  ><img
+                    src="./assets/nav-icons/Footer-icon4.svg"
+                    alt="twitter"
+                    class="footer-icon"
+                /></a>
+              </li>
+            </ul>
           </div>
```

Comment:
- [ ] To stick to the Figma template design I think it would be better if you add the horizontal button after below icons and style it correctly
![Screenshot from 2023-02-21 17-40-47](https://user-images.githubusercontent.com/83042742/220391340-80cc22dc-821c-413f-b4ad-851810592037.png)

Reply:
@tresorsawasawa Thank you so much for your review. As for this horizontal bar below, I do not think it is to be added to the design because it is part of an iPhone's design, and just as the time, wifi and battery above shouldn’t be part of the design, so is this not meant to be part of it. Kindly. correct me if I’m wrong.